app/controllers/doorkeeper/openid_connect/discovery_controller.rb

```diff
@@ -89,7 +89,7 @@ def keys_response
 
         {
           keys: [
-            signing_key.slice(:kty, :kid, :e, :n).merge(
+            signing_key.slice(:kty, :kid, :e, :n, :x, :y, :crv).merge(
```

Comment:
these short var names are worth a comment describing them in my opinion.
or at least a link to where they are defined